node-installer/internal/constants/configuration-qemu-tdx.toml

```diff
@@ -0,0 +1,61 @@
+# Minimized list, inactive options removed.
+# upstream source: https://github.com/kata-containers/kata-containers/blob/0f2a4d202e90b39b50074725b2cfe9c3088a4e20/src/runtime/config/configuration-qemu-tdx.toml.in
+[hypervisor.qemu]
+path = "/usr/bin/qemu-system-x86_64"
+kernel = "/opt/kata/share/kata-containers/vmlinuz-confidential.container"
+image = "/opt/kata/share/kata-containers/kata-containers-confidential.img"
+machine_type = "q35"
+tdx_quote_generation_service_socket_port = 4050
+rootfs_type="erofs"
+confidential_guest = true
+enable_annotations = ["enable_iommu", "virtio_fs_extra_args", "kernel_params", "default_vcpus", "default_memory", "default_vcpus"]
```

Comment:
Why does this list `default_vcpus` twice?

Reply:
Not sure, it's also the case in the [upstream config](https://github.com/kata-containers/kata-containers/blob/81d23a18651d31704beba67c0bc7d58781931f1f/src/runtime/Makefile#L756), but it shouldn't hurt to remove it.